ROADMAP.md

```diff
@@ -1,56 +1,109 @@
 # Roadmap
 
-This document provides information on OpenEBS development in current and upcoming releases. Community and contributor involvement is vital for successfully implementing all desired items for each release. We hope that the items listed below will inspire further engagement from the community to keep OpenEBS progressing and shipping exciting and valuable features.
+This Roadmap defines OpenEBS features and capabilities that are in current development and may be incldued in upcoming releases.<BR>
```

Comment:
included*

Reply:
I see resolved but the typo is still here @orville-wright, did you perhaps forget to git push?